src/main/java/com/powsybl/openloadflow/network/util/ActivePowerDistribution.java

```diff
@@ -94,6 +100,22 @@ public static ActivePowerDistribution create(LoadFlowParameters.BalanceType bala
             default:
                 throw new UnsupportedOperationException("Unknown balance type mode: " + balanceType);
         }
-        return new ActivePowerDistribution(step);
+        return step;
+    }
+
+    public static Step getStep(LoadFlowParameters.BalanceType balanceType) {
```

Comment:
This is a very dangerous design. How someone is using it by calling getStep wan now it is only for DC purpose?

Reply:
Ok, we will follow your advice.